spec/features/apply_to_adopt_spec.rb

```diff
@@ -213,6 +214,7 @@
     fill_in('adopter_adoption_app_attributes_how_did_you_hear', with: 'Google and Craigslist')
     fill_in('adopter_adoption_app_attributes_ready_to_adopt_dt', with: '2012-12-13')
     choose('adopter_adoption_app_attributes_is_ofage_true')
+    choose('adopter_adoption_app_attributes_has_family_under_18_true')
     choose('adopter_adoption_app_attributes_house_type_own')
```

Comment:
Add a ```expect(page).to have_content('Has Family Under 18')```  toward the bottom of the test to verify that the value is showing up in the application view correctly.